README.md

```diff
@@ -1,3 +1,16 @@
 # Machine learning for Project Cognoma
 
 This repository hosts machine learning code and discussion (see [Issues](https://github.com/cognoma/machine-learning/issues)) for Project Cognoma.
+
+## Environment
+
+This repository uses [conda](http://conda.pydata.org/docs/ "Conda package management system and environment management system documentation") to manage its environment and install packages. If you don't have conda installed on your system, you can [download it here](http://conda.pydata.org/miniconda.html "Miniconda Homepage"). You can install the Python 2 or 3 version of Miniconda (or Anaconda), which determines the Python version of your root environment. Since we create a dedicated environment for this project, named `cognoma-machine-learning` whose explicit dependencies are specified in [`environment.yml`](environment.yml), the version of your root environment will not be relevant.
+
+With conda, you can create the `cognoma-machine-learning` environment using:
```

Comment:
Add note here about forking/cloning repository before creating the environment - at least something mentioning the specific `environment.yml` must be in directory